### PR TITLE
Set up store and wire it to text input component

### PR DESCRIPTION
**Pull request by bereket-WMDE**
Bug: T264318

### CODE REVIEW

File: src/components/QueryBuilder.vue [hunk 1]
```diff
@@ -22,9 +25,25 @@
...
 import TextInput from '@wmde/wikit-vue-components/src/components/TextInput.vue';
+import { mapState } from 'vuex';
 
 export default Vue.extend( {
 	name: 'QueryBuilder',
+
+	data() {
+		return {
+			textInputValue: '',
+			textInputProperty: ''
```
Comment (jakobw) [inline src/components/QueryBuilder.vue]: Neither of these are actually used. I think the `textInputProperty` one can just be deleted, and the value TextInput should actually read from the vuex store in the same way that the property one does, i.e. mapped it to a computed property from the store state.

**Review by jakobw — CHANGES_REQUESTED** [review]
Looks good! I only found one small thing that we should take care of before merging.

**Review by jakobw — APPROVED** [review]
Needs some commit history cleanup, but otherwise looks great! Woo! :rocket:

**Comment by guergana** [comment]
Looks good to me too. Just resolve the conflicts. Looks like it needs a rebase

**Review by jakobw — APPROVED** [review]
:100:

**Comment by bereket-WMDE** [comment]
> Looks good! I only found one small thing that we should take care of before merging.

already fixed the merging issue

**Pull request merged**